src/published-data/published-data.service.ts

```diff
@@ -44,8 +45,17 @@ export class PublishedDataService {
     private readonly httpService: HttpService,
     @Inject(REQUEST)
     private request: Request,
+    private configService: ConfigService,
   ) {}
 
+  async getConfig(): Promise<Record<string, unknown> | null> {
+    const config =
+      this.configService.get<Record<string, unknown>>("publishedDataConfig") ||
+      null;
+
+    return config;
+  }
+
```

Comment:
It'd be better to generate one from example if no publishedDataConfig json is provided 
Currently it fails silently when no publishedDataConfig is provided